src/api/consumer/lib/untrack.js

```diff
@@ -0,0 +1,40 @@
+/** @flow */
+import includes from "lodash.includes";
+import R from "ramda";
+import BitMap from "../../../consumer/bit-map";
+import {loadConsumer, Consumer} from "../../../consumer";
+import ComponentsList from "../../../consumer/component/components-list";
+
+
+export default async function untrack(componentIds: string[]): Promise<Object> {
+
+  const untrackedComponents = [];
+  const consumer: Consumer = await loadConsumer();
+  const bitMap = await BitMap.load(consumer.getPath());
+  const componentsList = new ComponentsList(consumer);
+  const newComponents = await componentsList.listNewComponents(true);
+  const stagedComponents = await componentsList.listExportPendingComponents();
```

Comment:
get the list export / list modified is expensive operations and you don't really need them.
If a component is not missing and not in the new list it should just be in the unRemovable